packages/integration_test/lib/integration_test_driver_extended.dart

```diff
@@ -0,0 +1,84 @@
+import 'dart:async';
+import 'dart:io';
+
+import 'common.dart';
+
+import 'package:flutter_driver/flutter_driver.dart';
+
+/// Example Integration Test which can also run WebDriver action depending on
+/// the requests coming from the test methods.
+Future<void> integrationDriver() async {
+  final FlutterDriver driver = await FlutterDriver.connect();
+
+  // Test states that it's waiting on web driver commands.
+  String jsonResponse = await driver.requestData(
```

Comment:
No this is not a breaking change. All existing tests are passing. This statement is false: " API for integration test purposes, because the developer may need to use requestData for their test-specific needs. If they try to use it, either they will break integration_test functionality, or integration_test will break their code." Please read the code here: https://github.com/flutter/plugins/blob/95476a5bb20d04ac85f87315c84bdaeee3962a1c/packages/integration_test/lib/_driver_commands_web.dart#L77

"Instead, integration_test should establish its own dedicated 2-way communication channel with the app side and use it exclusively to implement this protocol." --> This is exactly what is happening today.